HOWTO-it.md

```diff
@@ -0,0 +1,17 @@
+Leggilo in altre lingue: [English](HOWTO.md), [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [Italiano](HOWTO-it.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md)
```

Comment:
:information_source: Maybe link also the other translation not merged yet?

- #5684 German (at this moment conficted with Russian)